components/navigation/NavBar.js

```diff
@@ -127,7 +168,19 @@ export default function NavBar({
               <IconLoupe />
             </SearchButton>
 
-            <GithubButton text="Star on GitHub" href="https://github.com/asyncapi/spec" className="py-2 ml-2" inNav="true" />
+            <LanguageSelect
+              options={uniqueLangs}
+              onChange={(value) => {
+                setLang(value.toLowerCase());
+                changeLanguage(value.toLowerCase());
+              }}
+              className=""
+              selected={i18n.language.toLocaleUpperCase()}
+            />
```

Comment:
We can have that code here, but please comment it now, we will enable that in last PR :)